.test-defs/CreateShoot.yaml

```diff
@@ -0,0 +1,14 @@
+kind: TestDefinition
+metadata:
+  name: create-shoot
+spec:
+  owner: Gardener-Team
+  description: Tests the creation of a shoot.
+
+  activeDeadlineSeconds: 3600
+  labels: []
+
+  command: [bash, -c]
+  args: ["/tm/setup github.com/gardener gardener && go run $GOPATH/src/github.com/gardener/gardener/.test-defs/cmd/create-shoot"]
```

Comment:
In another file you write

```yaml
  - >-
    /tm/setup github.com/gardener gardener &&
    go test $GOPATH/src/github.com/gardener/gardener/test/integration/shoots/applications
    -ginkgo.v -ginkgo.progress
    -kubeconfig=$TM_KUBECONFIG_PATH/gardener.config
    -shootName=$SHOOT_NAME
    -shootNamespace=$PROJECT_NAMESPACE
```

-> Better readability. Can you do the same here, please?